### ISSUE TITLE
DESCRIPTION file error

### ISSUE THREAD

**Issue by thibautjombart** [reconhub/nomad#19]
Package seems broken by the recent changes (wrong `DESCRIPTION` file?):
```
> devtools::install_github("reconhub/nomad", upgrade = FALSE)

Downloading GitHub repo reconhub/nomad@master
✔  checking for file ‘/tmp/Rtmpr6w0bQ/remotes52f563ea753b/reconhub-nomad-5666c6b/DESCRIPTION’
─  preparing ‘nomad’:
─  checking DESCRIPTION meta-information ...Warning in c.person(person("Rich", "FitzJohn", email = "rich.fitzjohn@gmail.com",  :
     method is only applicable to ‘person’ objects
    OK
   Warning in c.person(person("Rich", "FitzJohn", email = "rich.fitzjohn@gmail.com",  :
     method is only applicable to ‘person’ objects
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘nomad_0.0.1.tar.gz’
   
Installing package into ‘/home/thibaut/R/x86_64-pc-linux-gnu-library/3.5’
(as ‘lib’ is unspecified)
* installing *source* package ‘nomad’ ...
Error in c.person(person("Rich", "FitzJohn", email = "rich.fitzjohn@gmail.com",  : 
  (converted from warning) method is only applicable to ‘person’ objects
ERROR: installing package DESCRIPTION failed for package ‘nomad’
* removing ‘/home/thibaut/R/x86_64-pc-linux-gnu-library/3.5/nomad’
Error in i.p(...) : 
  (converted from warning) installation of package ‘/tmp/Rtmpr6w0bQ/file52f57e168bf4/nomad_0.0.1.tar.gz’ had non-zero exit status
> 
```

The R version used is:
```
> R.version
               _                           
platform       x86_64-pc-linux-gnu         
arch           x86_64                      
os             linux-gnu                   
system         x86_64, linux-gnu           
status                                     
major          3                           
minor          5.3                         
year           2019                        
month          03                          
day            11                          
svn rev        76217                       
language       R                           
version.string R version 3.5.3 (2019-03-11)
nickname       Great Truth                 
```

**This is critical - we need fixing ASAP.**

**Comment by zkamvar** [comment]
Should be fixed in #20.

``` r
remotes::install_github("reconhub/nomad") # error
#> Downloading GitHub repo reconhub/nomad@master
#> 
#>   
   checking for file ‘/tmp/RtmpgZzPnv/remotes479d63b58205/reconhub-nomad-5666c6b/DESCRIPTION’ ...
  
✔  checking for file ‘/tmp/RtmpgZzPnv/remotes479d63b58205/reconhub-nomad-5666c6b/DESCRIPTION’
#> 
  
─  preparing ‘nomad’:
   checking DESCRIPTION meta-information ...
─  checking DESCRIPTION meta-information ...Warning in c.person(person("Rich", "FitzJohn", email = "rich.fitzjohn@gmail.com",  :
#>      method is only applicable to ‘person’ objects
#> 
  
    OK
#> 
  
   Warning in c.person(person("Rich", "FitzJohn", email = "rich.fitzjohn@gmail.com",  :
#>      method is only applicable to ‘person’ objects
#> 
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘nomad_0.0.1.tar.gz’
#> Installing package into '/home/zkamvar/Documents/RLibrary'
#> (as 'lib' is unspecified)
#> Adding 'nomad_0.0.1_R_x86_64-pc-linux-gnu.tar.gz' to the cache
```
```r
remotes::install_github("reconhub/nomad#20") # works
#> Downloading GitHub repo reconhub/nomad@fix-19
#> 
#>   
   checking for file ‘/tmp/RtmpgZzPnv/remotes479d354a40ed/reconhub-nomad-431eecf/DESCRIPTION’ ...
✔  checking for file ‘/tmp/RtmpgZzPnv/remotes479d354a40ed/reconhub-nomad-431eecf/DESCRIPTION’
─  preparing ‘nomad’:
   checking DESCRIPTION meta-information ...
✔  checking DESCRIPTION meta-information
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories  
─  building ‘nomad_0.0.1.tar.gz’

#> Installing package into '/home/zkamvar/Documents/RLibrary'
#> (as 'lib' is unspecified)
```

<sup>Created on 2019-04-24 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1)</sup>

**Comment by thibautjombart** [comment]
Awesome, thanks :)
